### PR TITLE
Fluffcomputers

### PR DESCRIPTION
**Pull request by jakmak3**
This adds in several new computers that do litterally nothing except give a access denied message. The only purpose for these computers is for vaults/away missions for immhulsions purposes. When jwmturners mapping PR is closed/merged i will make a pr to replace some of the empty computer consoles in the white ship with these.

### CODE REVIEW

File: code/game/machinery/computer/fluffcomputer.dm [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+/obj/machinery/computer/unusable_shuttle_control_1/attack_paw(var/mob/user as mob)
+
+    return src.attack_hand(user)
+    return
+
+/obj/machinery/computer/unusable_shuttle_control_1/attack_hand(var/mob/user as mob)
+    say("Access Denied")
+
+/obj/machinery/computer/unusable_shuttle_control_1/emag(mob/user as mob)
+    to_chat(user, "You hold the cryptographic sequencer up to the ID scanner and nothing happens. Guess this only works")
```
Comment (kol1th) [inline code/game/machinery/computer/fluffcomputer.dm]: missed a word, oopsies

File: code/game/machinery/computer/fluffcomputer.dm [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+/obj/machinery/computer/unusable_shuttle_control_2 /*fluff shuttle control 2 */
+    name = "Shuttle Control"
+    desc = "A code locked console that controls the ship. Doesnt look like you will be flying the ship anytime soon."
```
Comment (kol1th) [inline code/game/machinery/computer/fluffcomputer.dm]: Doesn't

File: code/game/machinery/computer/fluffcomputer.dm [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+/obj/machinery/computer/unusable_shuttle_control_2/attack_hand(var/mob/user as mob)
+    say("Access Denied")
+
+/obj/machinery/computer/unusable_shuttle_control_2/emag(mob/user as mob)
+    to_chat(user, "You hold the cryptographic sequencer up to the ID scanner and nothing happens. Guess this only works with NT technology")
+
+
+/obj/machinery/computer/unusable_shuttle_engine_control /*fluff shuttle console 3 */
+    name = "Engine Control"
+    desc = "A code locked console that controls the ships engines and power systems."
```
Comment (kol1th) [inline code/game/machinery/computer/fluffcomputer.dm]: ship's

File: code/game/machinery/computer/fluffcomputer.dm [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+/obj/machinery/computer/unusable_shuttle_engine_control/emag(mob/user as mob)
+    to_chat(user, "You hold the cryptographic sequencer up to the ID scanner and nothing happens. Guess this only works with NT technology ")
+
+
+ /* the next 2 are for the white ship */
+
+/obj/machinery/computer/whiteship_console_1  /*console 1*/
+    name = "Starmap"
+    desc = "A console with a map of the local area. Just by looking at this thing you can tell its years out of date and is too old to be used effectivly"
```
Comment (kol1th) [inline code/game/machinery/computer/fluffcomputer.dm]: it's or it is

File: code/game/machinery/computer/fluffcomputer.dm [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+    icon_state = "comm_serv"
+    circuit = "/obj/item/weapon/circuitboard/whiteship_console_1"
+
+    light_color = LIGHT_COLOR_GREEN
+
+/obj/machinery/computer/whiteship_console_1/attack_ai(var/mob/user as mob)
+        to_chat(user, "The console is so old that it cannot be remotely controlled by sillicons")
```
Comment (kol1th) [inline code/game/machinery/computer/fluffcomputer.dm]: add a period, and silicons

File: code/game/machinery/computer/fluffcomputer.dm [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+/obj/machinery/computer/whiteship_console_1/attack_paw(var/mob/user as mob)
+
+    return src.attack_hand(user)
+    return
+
+/obj/machinery/computer/whiteship_console_1/attack_hand(var/mob/user as mob)
+    say("Access Denied") /* maybe if it displayed a static image of like a outdated map of the local area for lore purposes? */
+
+/obj/machinery/computer/whiteship_console_1/emag(mob/user as mob)
+	to_chat(user, "The console must be so old that not even the cryptographic sequencer is compatible with it ")
```
Comment (kol1th) [inline code/game/machinery/computer/fluffcomputer.dm]: Maybe something like `"The cryptographic sequencer causes some lights on the console to light up before emitting a dull buzzing noise."`

File: code/game/machinery/computer/fluffcomputer.dm [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+
+
+/obj/machinery/computer/whiteship_console_2  /*console 2*/
+    name = "Engine Control"
+    desc = "A console displaying the status of the ships engines."
```
Comment (kol1th) [inline code/game/machinery/computer/fluffcomputer.dm]: ship's

File: code/game/machinery/computer/fluffcomputer.dm [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+    icon_state = "engine1"
+    circuit = "/obj/item/weapon/circuitboard/whiteship_console_2"
+
+    light_color = LIGHT_COLOR_BLUE
+
+/obj/machinery/computer/whiteship_console_2/attack_ai(var/mob/user as mob)
+        to_chat(user, "The console is so old that it cannot be remotely controlled by sillicons")
```
Comment (kol1th) [inline code/game/machinery/computer/fluffcomputer.dm]: same as before

File: code/game/machinery/computer/fluffcomputer.dm [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+/obj/machinery/computer/whiteship_console_2/attack_paw(var/mob/user as mob)
+
+    return src.attack_hand(user)
+    return
+
+/obj/machinery/computer/whiteship_console_2/attack_hand(var/mob/user as mob)
+    say("ERROR FUEL RESERVE EMPTY. SHUTTING DOWN") /*needs a better message for why it doesnt work. Taking suggestions */
+
+/obj/machinery/computer/whiteship_console_2/emag(mob/user as mob)
+    to_chat(user, "The console must be so old that not even the cryptographic sequencer is compatible with it ")
```
Comment (kol1th) [inline code/game/machinery/computer/fluffcomputer.dm]: same as above

File: code/game/machinery/computer/fluffcomputer.dm [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+/obj/machinery/computer/whiteship_console_2/attack_ai(var/mob/user as mob)
+        to_chat(user, "The console is so old that it cannot be remotely controlled by sillicons")
+
+/obj/machinery/computer/whiteship_console_2/attack_paw(var/mob/user as mob)
+
+    return src.attack_hand(user)
+    return
+
+/obj/machinery/computer/whiteship_console_2/attack_hand(var/mob/user as mob)
+    say("ERROR FUEL RESERVE EMPTY. SHUTTING DOWN") /*needs a better message for why it doesnt work. Taking suggestions */
```
Comment (kol1th) [inline code/game/machinery/computer/fluffcomputer.dm]: `WARNING: FUEL RESERVES LOW. ENGINES SHUTTING DOWN.` maybe?

**Comment by Zth--** [comment]
Images or gifs _please_

File: code/game/machinery/computer/buildandrepair.dm [hunk 11]
```diff
@@ -269,6 +269,66 @@
 	build_path = "/obj/machinery/computer/stacking_unit"
 	origin_tech = Tc_PROGRAMMING + "=2;" + Tc_MATERIALS + "=2"
 
+/obj/item/weapon/circuitboard/unusable_communication_relay
+	name = "Circut Board (Communications relay)"
```
Comment (PJB3005) [inline code/game/machinery/computer/buildandrepair.dm:273]: All of these need to not be capitalized.

File: code/game/machinery/computer/fluffcomputer.dm [hunk 12]
```diff
@@ -0,0 +1,141 @@
+/* this entire thing is a non player operatable computer that just gives a access denied message or something like that when you try to use it.
+It does nothing and only serves to look good and for immhulsions. intended for vaults away missions and things of that sort.
+This is not related to actual functional consoles in any way */
+
+/obj/machinery/computer/unusable_communication_relay
+    name = "Communications Relay"
```
Comment (PJB3005) [inline code/game/machinery/computer/fluffcomputer.dm:6]: Remove capitalization.

File: code/game/machinery/computer/fluffcomputer.dm [hunk 13]
```diff
@@ -0,0 +1,141 @@
...
+
+    light_color = LIGHT_COLOR_BLUE
+
+/obj/machinery/computer/unusable_communication_relay/attack_ai(var/mob/user as mob)
+        to_chat(user, "The systems firewall prevents you from accessing it.")
+
+/obj/machinery/computer/unusable_communication_relay/attack_paw(var/mob/user as mob)
+
+    return src.attack_hand(user)
+    return
```
Comment (PJB3005) [inline code/game/machinery/computer/fluffcomputer.dm:19]: This is the worst incident of trailing return I've seen, ever!

File: code/game/machinery/computer/fluffcomputer.dm [hunk 14]
```diff
@@ -0,0 +1,141 @@
...
+/obj/machinery/computer/unusable_communication_relay
+    name = "Communications Relay"
+    desc = "A code locked console that you do not know how to use. Probably doesnt do anything interesting."
+    icon_state = "comm"
+    circuit = "/obj/item/weapon/circuitboard/unusable_communication_relay"
+
+    light_color = LIGHT_COLOR_BLUE
+
+/obj/machinery/computer/unusable_communication_relay/attack_ai(var/mob/user as mob)
+        to_chat(user, "The systems firewall prevents you from accessing it.")
```
Comment (PJB3005) [inline code/game/machinery/computer/fluffcomputer.dm:14]: Is GitHub screwing with me or is this line indented twice?

File: code/game/machinery/computer/fluffcomputer.dm [hunk 15]
```diff
@@ -0,0 +1,141 @@
...
+
+/obj/machinery/computer/unusable_communication_relay/attack_paw(var/mob/user as mob)
+
+    return src.attack_hand(user)
+    return
+
+/obj/machinery/computer/unusable_communication_relay/attack_hand(var/mob/user as mob)
+    say("Access Denied")
```
Comment (PJB3005) [inline code/game/machinery/computer/fluffcomputer.dm:22]: Make this check for whether the machine actually has power and such.

File: code/game/machinery/computer/fluffcomputer.dm [hunk 16]
```diff
@@ -0,0 +1,141 @@
...
+
+
+/obj/machinery/computer/unusable_communication_relay/emag(mob/user as mob)
+    to_chat(user, "You hold the cryptographic sequencer up to the ID scanner and nothing happens. Guess this only works with NT technology ")
```
Comment (PJB3005) [inline code/game/machinery/computer/fluffcomputer.dm:26]: Make this check for whether the machine actually has power and such.

Also, you forgot a period.

File: code/game/machinery/computer/fluffcomputer.dm [hunk 17]
```diff
@@ -0,0 +1,141 @@
...
+
+/obj/machinery/computer/unusable_shuttle_control_1 /*fluff shuttle console 1 */
+    name = "Shuttle Control"
```
Comment (PJB3005) [inline code/game/machinery/computer/fluffcomputer.dm:29]: Remove capitalization.

Comment (PJB3005) [reply code/game/machinery/computer/fluffcomputer.dm:29]: Also, same issues that plague the other one.

**Review by PJB3005 — CHANGES_REQUESTED** [review]

**Comment by ComicIronic** [comment]
Put these under an unusable/ type and reduce the copypaste.

File: code/game/machinery/computer/fluffcomputer.dm (hunk 14 above)
Comment (kol1th) [reply code/game/machinery/computer/fluffcomputer.dm:14]: It's indented twice

File: code/game/machinery/computer/buildandrepair.dm (hunk 11 above)
Comment (kol1th) [reply code/game/machinery/computer/buildandrepair.dm:273]: also 

> circut

File: code/game/machinery/computer/fluffcomputer.dm (hunk 13 above)
Comment (ghost) [reply code/game/machinery/computer/fluffcomputer.dm:19]: oh my god that's amazing

File: code/game/machinery/computer/fluffcomputer.dm [hunk 18]
```diff
@@ -0,0 +1,141 @@
...
+    return
+
+/obj/machinery/computer/unusable_communication_relay/attack_hand(var/mob/user as mob)
+    say("Access Denied")
+
+
+/obj/machinery/computer/unusable_communication_relay/emag(mob/user as mob)
+    to_chat(user, "You hold the cryptographic sequencer up to the ID scanner and nothing happens. Guess this only works with NT technology ")
+
+/obj/machinery/computer/unusable_shuttle_control_1 /*fluff shuttle console 1 */
```
Comment (ghost) [inline code/game/machinery/computer/fluffcomputer.dm:28]: why use /* */ for a single line?

File: code/game/machinery/computer/fluffcomputer.dm [hunk 19]
```diff
@@ -0,0 +1,141 @@
...
+
+    light_color = LIGHT_COLOR_RED
+
+/obj/machinery/computer/unusable_shuttle_control_2/attack_ai(var/mob/user as mob)
+        to_chat(user, "The systems firewall prevents you from accessing it.")
+
+/obj/machinery/computer/unusable_shuttle_control_2/attack_paw(var/mob/user as mob)
+
+    return src.attack_hand(user)
+    return
```
Comment (ghost) [inline code/game/machinery/computer/fluffcomputer.dm:64]: OH NO HE DID IT AGAIN

File: code/game/machinery/computer/fluffcomputer.dm [hunk 20]
```diff
@@ -0,0 +1,141 @@
...
+
+    light_color = LIGHT_COLOR_RED
+
+/obj/machinery/computer/unusable_shuttle_engine_control/attack_ai(var/mob/user as mob)
+        to_chat(user, "The systems firewall prevents you from accessing it.")
+
+/obj/machinery/computer/unusable_shuttle_engine_control/attack_paw(var/mob/user as mob)
+
+    return src.attack_hand(user)
+    return
```
Comment (ghost) [inline code/game/machinery/computer/fluffcomputer.dm:87]: AND AGAIN

File: code/game/machinery/computer/fluffcomputer.dm [hunk 21]
```diff
@@ -0,0 +1,141 @@
...
+
+    light_color = LIGHT_COLOR_GREEN
+
+/obj/machinery/computer/whiteship_console_1/attack_ai(var/mob/user as mob)
+        to_chat(user, "The console is so old that it cannot be remotely controlled by silicons.")
+
+/obj/machinery/computer/whiteship_console_1/attack_paw(var/mob/user as mob)
+
+    return src.attack_hand(user)
+    return
```
Comment (ghost) [inline code/game/machinery/computer/fluffcomputer.dm:112]: AND AGAAAAIN

File: code/game/machinery/computer/fluffcomputer.dm (hunk 15 above)
Comment (Kurfursten) [reply code/game/machinery/computer/fluffcomputer.dm:22]: don't these typically use the warning span?

**Comment by FudgePucker** [comment]
An idea for fluff might be a way to hack it by hand, to give fluff text, or a way to output vault coordinates.

Goons solarium is an example of how you can use puzzles and mysteries to add depth and atmosphere to areas in space rather than just having them be loot zones. The telescience adventures on goon is one of the few parts I liked about that code base (fuck their mining and incomplete material system). I have rarely gotten that sense of emersion that I had going though those areas for the first time, looking around every corner for traps, reading logs left behind.

**Comment by jakmak3** [comment]
OK FUCK THIS IM MAKING A NEW PR GIT HATH  REPREHENSIBLY FUCKED ON MY END

**Pull request closed without merging**